tests/Agent/IntegrationTests/IntegrationTests/InfiniteTracing/InfiniteTracingTests.cs

```diff
@@ -21,17 +21,16 @@ public abstract class InfiniteTracingTestsBase<TFixture> : NewRelicIntegrationTe
         public InfiniteTracingTestsBase(TFixture fixture, ITestOutputHelper output) : base(fixture)
         {
             _fixture = fixture;
-            _fixture.SetTimeout(System.TimeSpan.FromMinutes(2));
+            _fixture.SetTimeout(TimeSpan.FromMinutes(3));
             _fixture.TestLogger = output;
 
             _fixture.AddCommand($"InfiniteTracingTester StartAgent");
 
-            _fixture.AddCommand("RootCommands DelaySeconds 5"); // give the agent time to warm up
+            // Give the agent time to warm up... If we send a span too soon, it will be sent via DT (span_event_data) instead of 8T (gRPC)
+            _fixture.AddCommand("RootCommands DelaySeconds 15"); 
```

Comment:
Feature request: We need an elegant way to wait for log lines interleaved with the ConsoleMF commands array